services/distribution/src/main/resources/application.yaml

```diff
@@ -225,26 +225,27 @@ services:
       dgc-directory: ${DIGITAL_GREEN_CERTIFICATE_DIRECTORY:ehn-dgc}
       valuesets-file-name: ${DIGITAL_GREEN_CERTIFICATE_VALUESETS_FILE_NAME:value-sets}
       supported-languages: ${DIGITAL_GREEN_CERTIFICATE_SUPPORTED_LANGUAGES:DE, EN, BG, PL, RO, TR}
+      booster-notification: ${BOOSTER_NOTIFICATION_DIRECTORY:booster-notification-rules}
       dsc-client:
-        public-key: ${DSC_PUBLIC_KEY:MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAETHfi8foQF4UtSNVxSFxeu7W+gMxdSGElhdo7825SD3Lyb+Sqh4G6Kra0ro1BdrM6Qx+hsUx4Qwdby7QY0pzxyA==}
-        base-url: ${DSC_BASE_PATH:https://de.dscg.ubirch.com}
+        public-key: ${DSC_PUBLIC_KEY:MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEIxHvrv8jQx9OEzTZbsx1prQVQn/3ex0gMYf6GyaNBW0QKLMjrSDeN6HwSPM0QzhvhmyQUixl6l88A7Zpu5OWSw==}
```

Comment:
What is the background of this change?

Reply:
This was before used to connect to the PRD dsc client from the local environment. But since we had the problem in PRD to connect to the dsc because of the whitelisting IPs, we have the same problem from the local env. So we changed this to the dev landscape so we can run the distribution against dev landscape from DSC